[BUILD] compile error in oiioutils.cpp

<!-- ⚠ Do not delete this issue template! ⚠ -->

**Commit Hash** <!-- 8 character string of letters/numbers in title bar or Help > About dialog (e.g. 3ea173c9) -->
afbd3ada3696bdef608c547bd1dd5d3a7e806e3b

**Platform** <!-- e.g. Windows 10, Ubuntu 20.04 or macOS 10.15 -->
Gentoo

**Summary**
I get a compile error as follows:
```
FAILED: app/CMakeFiles/olive-editor.dir/common/oiioutils.cpp.o 
/usr/bin/c++  -DAPPVERSION=\"0.2.0\" -DGITHASH=\"afbd3ada\" -DQT_CONCURRENT_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_GUI_LIB -DQT_MULTIMEDIA_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_SVG_LIB -DQT_WIDGETS_LIB -Iapp -I../app -Iapp/olive-editor_autogen/include -I/usr/include/OpenEXR -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtCore -isystem /usr/lib64/qt5/mkspecs/linux-g++ -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtMultimedia -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtOpenGL -isystem /usr/include/qt5/QtSvg -isystem /usr/include/qt5/QtConcurrent -Werror -Wuninitialized -pedantic-errors -Wall -Wextra -Wno-unused-parameter -Wshadow -fPIC -std=c++11 -MD -MT app/CMakeFiles/olive-editor.dir/common/oiioutils.cpp.o -MF app/CMakeFiles/olive-editor.dir/common/oiioutils.cpp.o.d -o app/CMakeFiles/olive-editor.dir/common/oiioutils.cpp.o -c ../app/common/oiioutils.cpp
../app/common/oiioutils.cpp: In static member function ‘static void olive::OIIOUtils::FrameToBuffer(const olive::Frame*, OpenImageIO_v1_8::ImageBuf*)’:
../app/common/oiioutils.cpp:33:41: error: ‘PixelFormat’ has not been declared
   33 |   int width_in_bytes = frame->width() * PixelFormat::BytesPerPixel(frame->format());
      |                                         ^~~~~~~~~~~
../app/common/oiioutils.cpp:42:23: error: passing ‘const olive::Frame’ as ‘this’ argument discards qualifiers [-fpermissive]
   42 |           frame->data() + i * frame->linesize_bytes(),
      |                       ^
In file included from ../app/common/oiioutils.h:27,
                 from ../app/common/oiioutils.cpp:21:
../app/codec/frame.h:103:9: note:   in call to ‘char* olive::Frame::data()’
  103 |   char* data()
      |         ^~~~
../app/common/oiioutils.cpp: In static member function ‘static void olive::OIIOUtils::BufferToFrame(OpenImageIO_v1_8::ImageBuf*, olive::Frame*)’:
../app/common/oiioutils.cpp:62:41: error: ‘PixelFormat’ has not been declared
   62 |   int width_in_bytes = frame->width() * PixelFormat::BytesPerPixel(frame->format());
      |  
```

**Additional Information / Output**

--- a/app/common/oiioutils.cpp
+++ b/app/common/oiioutils.cpp
@@ -19,7 +19,9 @@
 ***/
 
 #include "oiioutils.h"
-
+#if OIIO_VERSION < 20112
+#include "render/videoparams.h"
+#endif
 namespace olive {
 
 void OIIOUtils::FrameToBuffer(const Frame* frame, OIIO::ImageBuf *buf)
@@ -30,7 +32,7 @@ void OIIOUtils::FrameToBuffer(const Frame* frame, OIIO::ImageBuf *buf)
   //
   // See more: https://github.com/OpenImageIO/oiio/pull/2487
   //
-  int width_in_bytes = frame->width() * PixelFormat::BytesPerPixel(frame->format());
+  int width_in_bytes = frame->width() * VideoParams::GetBytesPerPixel(frame->format(), frame->channel_count());
 
   for (int i=0;i<buf->spec().height;i++) {
     memcpy(
@@ -39,7 +41,7 @@ void OIIOUtils::FrameToBuffer(const Frame* frame, OIIO::ImageBuf *buf)
 #else
           reinterpret_cast<char*>(buf->localpixels()) + i * buf->scanline_stride(),
 #endif
-          frame->data() + i * frame->linesize_bytes(),
+          frame->const_data() + i * frame->linesize_bytes(),
           width_in_bytes);
   }
 #else
@@ -59,7 +61,7 @@ void OIIOUtils::BufferToFrame(OIIO::ImageBuf *buf, Frame* frame)
   //
   // See more: https://github.com/OpenImageIO/oiio/pull/2487
   //
-  int width_in_bytes = frame->width() * PixelFormat::BytesPerPixel(frame->format());
+  int width_in_bytes = frame->width() * VideoParams::GetBytesPerPixel(frame->format(), frame->channel_count());
 
   for (int i=0;i<buf->spec().height;i++) {
     memcpy(frame->data() + i * frame->linesize_bytes(),

